[2024-04-13T01:28:32.799+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 12345678.insert_qdrant manual__2024-04-12T20:34:18.190143+00:00 [queued]>
[2024-04-13T01:28:32.805+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 12345678.insert_qdrant manual__2024-04-12T20:34:18.190143+00:00 [queued]>
[2024-04-13T01:28:32.805+0000] {taskinstance.py:2193} INFO - Starting attempt 2 of 3
[2024-04-13T01:28:32.814+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): insert_qdrant> on 2024-04-12 20:34:18.190143+00:00
[2024-04-13T01:28:32.818+0000] {standard_task_runner.py:60} INFO - Started process 2973 to run task
[2024-04-13T01:28:32.821+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', '12345678', 'insert_qdrant', 'manual__2024-04-12T20:34:18.190143+00:00', '--job-id', '108', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp5doihag5']
[2024-04-13T01:28:32.823+0000] {standard_task_runner.py:88} INFO - Job 108: Subtask insert_qdrant
[2024-04-13T01:28:32.859+0000] {task_command.py:423} INFO - Running <TaskInstance: 12345678.insert_qdrant manual__2024-04-12T20:34:18.190143+00:00 [running]> on host 5feba290cccc
[2024-04-13T01:28:32.910+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='12345678' AIRFLOW_CTX_DAG_ID='12345678' AIRFLOW_CTX_TASK_ID='insert_qdrant' AIRFLOW_CTX_EXECUTION_DATE='2024-04-12T20:34:18.190143+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-12T20:34:18.190143+00:00'
[2024-04-13T01:28:32.944+0000] {python.py:202} INFO - Done. Returned value was: {'status': 'success', 'message': 'Data inserted to Vector DB successfully'}
[2024-04-13T01:28:32.958+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=12345678, task_id=insert_qdrant, execution_date=20240412T203418, start_date=20240413T012832, end_date=20240413T012832
[2024-04-13T01:28:32.999+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-13T01:28:33.016+0000] {taskinstance.py:3309} INFO - 1 downstream tasks scheduled from follow-on schedule check
